src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = (a) => {
```

Comment:
Let's simplify a little
```suggestion
  return a => b => {
      counter++;
      
      if...
```